code/game/objects/items/weapons/melee/melee_misc.dm

```diff
@@ -40,20 +40,20 @@
 	materials = list(MAT_METAL = 1000)
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF // Theft targets should be hard to destroy
 
-/obj/item/melee/rapier/examine(mob/user)
+/obj/item/melee/sabre/examine(mob/user)
 	. = ..()
 	. += "<span class='notice'>The blade looks very well-suited for piercing armour.</span>"
 
-/obj/item/melee/rapier/examine_more(mob/user)
+/obj/item/melee/sabre/examine_more(mob/user)
 	. = ..()
 	. += "Swords are a traditional ceremonial weapon carried by commanding officers of many armies and navies, even long after firearms and laserarms rendered them obsolete. \
 	Despite having no roots in such traditions, Nanotrasen participates in them, as these trappings of old tradition help to promote the air of authority the company wishes for its captains to possess."
 	. += ""
-	. += "Whilst not intended to actually be used in combat, these ceremonial rapiers are in-fact quite functional, \
+	. += "Whilst not intended to actually be used in combat, the ceremonial blades issued by Nanotrasen are in-fact quite functional, \
 	able to both inflict grievous wounds on aggressors that get too close, whilst also elegantly parrying their blows (assuming the wielder is skilled with the blade). \
-	The thin, sharp point is also quite effective at at defeating even modern body armour, which tends to be designed to deal with ballistic and laser weapons rather than swords..."
+	The sharp edge is adept at hacking unarmoured targets, whilst the rigid tip is also quite effective at at defeating even modern body armour with thrusting attacks, as modern armour is generally designed to defeat ballistic and laser weapons rather than swords..."
```

Comment:
```suggestion
	The sharp edge is adept at hacking unarmored targets, whilst the rigid tip is also quite effective at at defeating even modern body armor with thrusting attacks, as modern armor is generally designed to defeat ballistic and laser weapons rather than swords..."
```